src/skins/vector/css/themes/_dark.scss

```diff
@@ -131,6 +131,9 @@ $progressbar-color: #000;
 }
 
 // markdown overrides:
+.mx_EventTile_content .markdown-body pre:hover {
+    border-color: #808080 !important; // inverted due to rules below
```

Comment:
should this not be some SCSS Var?

Reply:
Possibly. I'm not sure what the guidelines are for this, as it's an override from the base, but intentionally doesn't affect the light theme.

Reply:
this is a bit of a grey area (hah) - in this instance given all the other stuff nearby are hardcoded for now i'm happy to let it slip.